### ISSUE TITLE
Package Manager window name "Package Manager" is hardly visible in Black mode of High contrast.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#358]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode before launching the application.

1. Launch Xamarin Workbooks application and create a new workbook.
2. Navigate to File -> Add Package. Invoke "Add Package" . It will open "Package Manager" window.
3. Verify the Package Manager window name in black mode of High contrast.

### Expected Behavior
Window name should be clearly visible.
### Actual Behavior
"Package Manager" text is hardly visible in High contrast black mode.
Note: In high contrast white mode "Package Manager" text is clearly visible.
### User Impact
Low vision user will feel difficult to see the "Package Manager text"
### Screenshot
![13 hc b](https://user-images.githubusercontent.com/22956130/40724531-e099f5ee-643e-11e8-8d3a-615ca1480276.png)
![13 hc w](https://user-images.githubusercontent.com/22956130/40724532-e0c6afa8-643e-11e8-9ef4-f678d2ca0e0a.png)
![13 normal mode](https://user-images.githubusercontent.com/22956130/40724533-e0f1a05a-643e-11e8-95ff-30eaf6f89da5.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1
